<!DOCTYPE html>
<html>
<body>
<h1>Analytical tools and databases for immunological data</h1>

<h2>1. Modular machine learning platforms </h2>

<h4>1.1 SIMON using R, for transcriptome, flow cytometry, microbiome data, and meta data</h4>
  <p>A modular machine learning software for high dimensional biomedical data, it uses a graphical user interface, standardized pipelines for predictive modeling and automated machine learning.</p>
<p> <a href="https://www.cell.com/patterns/fulltext/S2666-3899(20)30242-7">  Cell Descriptor Paper 01/2021</a></p>
<p> <a href="https://www.jimmunol.org/content/early/2019/06/13/jimmunol.1900033">  Journal of Immunology Paper 2019</a></p>
<p> <a href="https://github.com/genular/simon-frontend/">  Github Page</a></p>

<h4>1.2 ImmuneML using Python, for adaptive immune receptor repertoire data and meta data</h4>

<p>A command-line tool and through an intuitive Galaxy web interface.</p>
<p> <a href="https://www.biorxiv.org/content/10.1101/2021.03.08.433891v3">  Preprint Paper 03/2021</a></p>
<p> <a href="https://docs.immuneml.uio.no/">  Documentation page</a></p>
<p> <a href="https://github.com/uio-bmi/immuneML">  Github page</a></p>

<h2>2 Descriptive Analysis tools for immune profiling V(D)J and transcriptome data</h2>
  
<h4>2.1 VDJView using R for 10X single cell V(D)J and GEX expression data</h4> 
<p>A wrapper for commonly used transcriptomic and receptor analysis packages (see below) in R shiny interface.</p>
<p> <a href="https://bmcmedgenomics.biomedcentral.com/articles/10.1186/s12920-020-0696-z">  Paper 2020</a></p>
<p> <a href="https://bitbucket.org/kirbyvisp/vdjview/src/master/">  Website</a></p> 

<h4>2.2 Immunarch using R for single cell and bulk including 10X, ArcherDX, MiXCR, and ImmunoSEQ</h4>
<p>An R package to perform repertoire-level and clonotype-level analyses.</p>
<p> <a href="https://immunarch.com/index.html">  Website</a></p>

<h4>2.3 Enclone using Rust from 10X genomics for 10X genomics V(D)J data</h4>
<p>A command-line tool to find and display clonotypes, and infer phylogenies.</p>
<p> <a href="https://10xgenomics.github.io/enclone/">  Website</a></p>

<h4>2.4 Scirpy using Python for single cell immune repertoires from 10X genomics and Smart-seq2</h4>
<p>Similar to the 2.1 and 2.2.</p>
<p> <a href="https://academic.oup.com/bioinformatics/article/36/18/4817/5866543">  Bioinformatics Paper 07/2020</a></p>
<p> <a href="https://icbi-lab.github.io/scirpy/">  Documentation page</a></p>
<p> <a href="https://github.com/icbi-lab/scirpy">  Github page</a></p>

<h4>2.5 Immcantation using both R and Python for single cell and bulk immune repertotire sequencing data</h4>
<p>The Immcantation framework provide a start-to-finish workflow of 6 packages for AIRR-seq datasets, beginning from raw reads to pre-processing, population structure determination, and repertoire analysis. 
<p> <a href="https://immcantation.readthedocs.io/en/stable/">  Documentation page</a></p>
</body>
</html>
